src/main/java/sirius/biz/jobs/params/CodeListParameter.java

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Couldn't we rely on a generic parameter here using CodeLists and the unique name of a code list instead of the id? This would spare us from creating separate implementations.

Reply:
CodeListParameter is now generic.